vivisect/parsers/elf.py

```diff
@@ -84,18 +84,20 @@ def makeRelocTable(vw, va, maxva, addbase, baseaddr):
     'amd64':'sysvamd64call',
 }
 
-def loadElfIntoWorkspace(vw, elf, filename=None):
+def loadElfIntoWorkspace(vw, elf, filename=None, arch=None, platform=None, filefmt='elf'):
```

Comment:
the cgc-specific stuff, and the 'elf' name never makes it to the Elf, they're acted upon in loadElfIntoWorkspace().  are you suggesting we make the Elf.Elf class identify these things, and have the loadElfIntoWorkspace() function draw them out through accessors?